site/content/docs/5.2/components/navbar.md

```diff
@@ -37,7 +37,7 @@ Navbars come with built-in support for a handful of sub-components. Choose from
 Here's an example of all the sub-components included in a responsive light-themed navbar that automatically collapses at the `lg` (large) breakpoint.
```

Comment:
Not due to this modification but navbar toggler icon isn't visible in light or dark mode:
![Screenshot 2022-11-25 at 11 59 39](https://user-images.githubusercontent.com/17381666/203970099-266b303d-04b2-41e6-973f-a2b23a251d49.png)
![Screenshot 2022-11-25 at 12 00 33](https://user-images.githubusercontent.com/17381666/203970248-6fe3d145-b512-4667-8e59-71126fc27d28.png)